external/js/cothority/src/byzcoin/proof.ts

```diff
@@ -383,6 +353,7 @@ class InclusionProof extends Message<InclusionProof> {
     leaf: LeafNode;
     empty: EmptyNode;
     nonce: Buffer;
+    protected _state: StateChangeBody;
```

Comment:
```suggestion
    protected _state: StateChangeBody | undefined;
```

Reply:
Hum, you mark it as protected but the field generating it (`leaf.value`) is publicicly modifiable; tag the latter as private?

Reply:
changed it to private.